__tests__/intergration.test.js

```diff
@@ -98,7 +98,29 @@ describe('GET /api/aricles', () => {
         });
       });
   });
+  test('should return a 200 containing all articles with the provided topic query', () => {
+    return request(app)
+    .get('/api/articles?filter_by=mitch')
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.articles.length).toBe(12);
+      expect(body.articles).toBeSortedBy('created_at', { descending: true });
+      body.articles.forEach((article) => {
+        expect(article.topic).toBe('mitch');
+      });
+    });
+  });
+  test('should return a 404 when the topic query does not exist', () => {
+    return request(app)
+    .get('/api/articles?filter_by=yellow')
+    .expect(404)
+    .then(({body}) => {
+      expect(body.msg).toBe('No results Found');
+    })
+  });
```

Comment:
I would recommend one additional test - what happens if we have a valid query, but there are no articles for it yet? At that point, it would still be a valid query, but we just would get back an empty array (almost like how we had to do an additional check for when an article just doesn't have any comments yet)